### ISSUE TITLE
libpq 10 build broken on Windows 

### ISSUE THREAD

**Issue by dvarrazzo** [psycopg/psycopg2#604]
Windows tests and packages using libpq 10 are broken by the commit https://github.com/postgres/postgres/commit/6da56f3f84d430671d5edd8f9336bd744c089e31 resulting in errors such as `NMAKE : fatal error U1052: file 'win32.mak' not found`

https://ci.appveyor.com/project/psycopg/psycopg2/build/2.x.155

**Comment by dvarrazzo** [comment]
Hi @jerickso I think we'll need your help on this.

I've tried fixing the appveyor file to work around the missing file, using the `build libpq` command. Changes are in the [fix-604 branch](https://github.com/psycopg/psycopg2/compare/libpq-10...fix-604?expand=1) currently resulting in [this build failure](https://ci.appveyor.com/project/psycopg/psycopg2/build/2.x.160). It smells like a static/dynamic lib mismatch to me...

Note that I have dropped `ENABLE_THREAD_SAFETY`, probably still needed needed but I have no idea how to pass it to the build system.

Thank you for any help!
